### PR TITLE
Feature/rifex 48/add rns subdomain

### PR DESCRIPTION
**Pull request by JONAF2103**
This PR introduces the ui to add a new RNS subdomain. This ui works as part of the detail of a domain, we have a menu where we can select subdomains as an option, then we have the page for subdomains where we can create and list subdomains. This PR is for the add funcionallity, here is a small demo:

demo-06-05-2020-jonathan-new-subdomain

This affects ui and service layers.

### CODE REVIEW

File: ui/app/rif/components/searchDomains.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ class SearchDomains extends Component {
 
   _handleKeyDown = (e) => {
     if (e.key === 'Enter') {
-      const typedDomain = e.target.value.toLowerCase();
+      const typedDomain = (e.target.value.toLowerCase() && e.target.value.toLowerCase().indexOf('.rsk') === -1) ? (e.target.value.toLowerCase() + '.rsk') : e.target.value.toLowerCase();
```
Comment (rorolopetegui) [inline ui/app/rif/components/searchDomains.js]: Maybe we can move this to utils.js or something better, what do you think about this?

File: ui/app/rif/pages/rns/subdomains/index.js [hunk 2]
```diff
@@ -74,7 +116,7 @@ class Subdomains extends Component {
           </button>
         </div>
         <div className="list">
-          {body}
+          {list}
```
Comment (rorolopetegui) [inline ui/app/rif/pages/rns/subdomains/index.js:119]: can we rename subDomainList? Or lstSubDomain?

**Review by rorolopetegui — CHANGES_REQUESTED** [review]
Just some minor changes, but in general it lgtm

File: ui/app/rif/pages/rns/subdomains/index.js (hunk 2 above)
Comment (JONAF2103) [reply ui/app/rif/pages/rns/subdomains/index.js:119]: I could but the component itself is subdomains so i guess is more than explicit right?

**Review by marcosmartinez7 — APPROVED** [review]
LGTM

**Review by rorolopetegui — APPROVED** [review]
lgtm

**Review by rorolopetegui — APPROVED** [review]
lgtm

**Pull request merged**